ipyparallel/apps/launcher.py

```diff
@@ -439,7 +439,7 @@ class MPILauncher(LocalProcessLauncher):
         help="The command line argument to the program."
     )
     n = Integer(1)
-    
+
```

Comment:
sorry about removing trailing whitespace, since it makes the diff bigger, I can remove that